framework/i18n/Formatter.php

```diff
@@ -772,18 +771,41 @@ private function formatDateTimeValue($value, $format, $type)
             }
             if (isset($this->_dateFormats[$format])) {
                 if ($type === 'date') {
-                    $formatter = new IntlDateFormatter($this->locale, $this->_dateFormats[$format], IntlDateFormatter::NONE, $timeZone, $this->calendar);
+                    $formatter = new IntlDateFormatter(
+                        $this->locale,
+                        $this->_dateFormats[$format],
+                        IntlDateFormatter::NONE,
+                        $timeZone,
+                        $this->calendar
+                    );
                 } elseif ($type === 'time') {
-                    $formatter = new IntlDateFormatter($this->locale, IntlDateFormatter::NONE, $this->_dateFormats[$format], $timeZone, $this->calendar);
+                    $formatter = new IntlDateFormatter(
+                        $this->locale,
+                        IntlDateFormatter::NONE,
+                        $this->_dateFormats[$format],
+                        $timeZone,
+                        $this->calendar
+                    );
                 } else {
-                    $formatter = new IntlDateFormatter($this->locale, $this->_dateFormats[$format], $this->_dateFormats[$format], $timeZone, $this->calendar);
+                    $formatter = new IntlDateFormatter(
+                        $this->locale,
+                        $this->_dateFormats[$format],
+                        $this->_dateFormats[$format],
+                        $timeZone,
+                        $this->calendar
+                    );
                 }
             } else {
-                $formatter = new IntlDateFormatter($this->locale, IntlDateFormatter::NONE, IntlDateFormatter::NONE, $timeZone, $this->calendar, $format);
-            }
-            if ($formatter === null) {
```

Comment:
Was this check lost?

Reply:
There is no way $formatter can be null here.

Reply:
OK.